src/server/test/config.ini

```diff
@@ -196,6 +196,11 @@ falcon_host = 127.0.0.1
 falcon_port = 1988
 falcon_path = /v1/push
 
+hotkey_analyse_time_interval = 1000000
```

Comment:
I think you can add unit for `hotkey_analyse_time_interval`. For example: `hotkey_analyse_time_interval_ms`